CHANGELOG.md

```diff
@@ -15,14 +15,21 @@
   Now you can compare everything that is `AsRef<str>` (such as `String` and `str`
   from the standard library) to uefi strings. Please head to the documentation of
   `EqStrUntilNul` to find out limitations and further information.
+- Added `BootServices::image_handle` to get the handle of the executing
+  image. The image is set automatically by the `#[entry]` macro; if a
+  program does not use that macro then it should call
+  `BootServices::set_image_handle`.
+- Added `BootServices::open_protocol_exclusive`. This provides a safe
+  and convenient subset of `open_protocol`.
```

Comment:
Maybe we should mention when `open_protocol_exclusive` is a good replacement for `open_protocol`? It works whenever we need to open a protocol on a resource which isn't shared with some other UEFI subsystem (e.g. not the right function for opening the graphics output protocol while stdout is also active).

Reply:
done